textlint/index.js

```diff
@@ -0,0 +1,186 @@
+		'abbr-within-parentheses': moduleInterop(require('textlint-rule-abbr-within-parentheses')),
+		'apostrophe': moduleInterop(require('textlint-rule-apostrophe')),
+		'common-misspellings': moduleInterop(require('textlint-rule-common-misspellings')),
+		'diacritics': moduleInterop(require('textlint-rule-diacritics')),
+		'doubled-spaces': moduleInterop(require('textlint-rule-doubled-spaces')),
+		'en-capitalization': moduleInterop(require('textlint-rule-en-capitalization')),
+		'en-max-word-count': moduleInterop(require('textlint-rule-en-max-word-count')),
+		'no-dead-link': moduleInterop(require('textlint-rule-no-dead-link')),
+		'no-empty-section': moduleInterop(require('textlint-rule-no-empty-section')),
+		'no-start-duplicated-conjunction': moduleInterop(require('textlint-rule-no-start-duplicated-conjunction')),
+		'terminology': moduleInterop(require('textlint-rule-terminology')),
+	},
+	rulesConfig: {
+		'diacritics': true,
+		'apostrophe': true,
+		'no-dead-link': true,
+		'doubled-spaces': true,
+		'no-empty-section': true,
+		'common-misspellings': true,
+		'abbr-within-parentheses': true,
+		'@textlint-rule/no-unmatched-pair': true,
+		'no-start-duplicated-conjunction': { interval: 2 },
+		'en-max-word-count': { max: 50 },
+		'en-capitalization': {
+			allowFigures: true,
+			allowLists: true,
+			allowWords: [
+				'bell',
+				'jit',
+				'jue',
+			],
+		},
+		'terminology': {
+			defaultTerms: false,
+			terms: [
+				'Airbnb',
+				'Android',
+				'AppleScript',
+				'AppVeyor',
+				'AVA',
+				'BrowserStack',
+				'Browsersync',
+				'Codecov',
+				'CodePen',
+				'CodeSandbox',
+				'DefinitelyTyped',
+				'EditorConfig',
+				'ESLint',
+				'Github',
+				'GraphQL',
+				'GraphiQL',
+				'iOS',
+				'Javascript',
+				'JetBrains',
+				'jQuery',
+				'LinkedIn',
+				'Lodash',
+				'MacBook',
+				'Markdown',
+				'OpenType',
+				'PayPal',
+				'PhpStorm',
+				'RubyMine',
+				'Sass',
+				'SemVer',
+				'TypeScript',
+				'UglifyJS',
+				'Wasm',
+				'WebAssembly',
+				'WebStorm',
+				'WordPress',
+				'Youtube',
+				[ 'Common[ .]js', 'CommonJS' ],
+				[ 'JSDocs?', 'JSDoc' ],
+				[ 'Node(?:js)?', 'Node.js' ],
+				[ 'React[ .]js', 'React' ],
+				[ 'SauceLabs', 'Sauce Labs' ],
+				[ 'StackOverflow', 'Stack Overflow' ],
+				[ 'styled ?components', 'styled-components' ],
+				[ 'HTTP[ /]2(?:\\.0)?', 'HTTP/2' ],
+				[ 'OS X', 'macOS' ],
+				[ 'Mac ?OS', 'macOS' ],
+				[ 'a npm', 'an npm' ],
+				'3D',
+				[ '3-D', '3D' ],
+				'Ajax',
+				'API',
+				[ 'API[\'’]?s', 'APIs' ],
+				'CSS',
+				'GIF',
+				'HTML',
+				'HTTPS',
+				'IoT',
+				'I/O',
+				[ 'I-O', 'I/O' ],
+				'JPEG',
+				'MIME',
+				'OK',
+				'PaaS',
+				'PDF',
+				'PNG',
+				'SaaS',
+				'URL',
+				[ 'URL[\'’]?s', 'URLs' ],
+				[ 'an URL', 'a URL' ],
+				[ 'wi[- ]?fi', 'Wi-Fi' ],
+				'ID',
+				[ 'id[\'’]?s', 'IDs' ],
+				[ 'backwards compatible', 'backward compatible' ],
+				[ 'she or he', 'they' ],
```

Comment:
The regex-replacing here feels like the perfect tool to expand on these un-binary-ing rules, so a few more added (please look over and confirm).

Reply:
Actually jack I need my gender neutral gliph. Jk jk Yah that seems good

Reply:
If you want you could add he or she and s(he)

Reply:
I see you did this already